tools/check_format_compatible.sh

```diff
@@ -63,6 +63,9 @@ scriptpath=`dirname ${BASH_SOURCE[0]}`
 test_dir=${TEST_TMPDIR:-"/tmp"}"/rocksdb_format_compatible_$USER"
 rm -rf ${test_dir:?}
 
+# Prevent 'make clean' etc. from wiping out test_dir
+export TEST_TMPDIR=$test_dir"/misc"
```

Comment:
Maybe I am missing something. `make clean` will do a `rm -rf $TEST_TMPDIR`. Will this prevent the dir from being wiped out?

Reply:
oh, I see. only `$test_dir`/misc will be removed.